SonarQube.Client/SonarQubeService.cs

```diff
@@ -103,12 +103,18 @@ public SonarQubeService(HttpMessageHandler messageHandler, string userAgent, ILo
         /// <param name="configure">Action that configures a type instance that implements TRequest.</param>
         /// <param name="token">Cancellation token.</param>
         /// <returns>Returns the result of the request invocation.</returns>
-        protected virtual async Task<TResponse> InvokeRequestAsync<TRequest, TResponse>(Action<TRequest> configure,
+        protected virtual Task<TResponse> InvokeRequestAsync<TRequest, TResponse>(Action<TRequest> configure,
```

Comment:
The `protected virtual` was added so the `MEFSonarQubeService` could override make sure all the web calls happened on the background thread. The override was here, because this was the low-level method.

It isn't anymore, so this method can go back to being `private`, and the new method can be `protected virtual`.